<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenAsDialog</class>
 <widget class="QDialog" name="OpenAsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>368</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Open As</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="inputFileGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Input File</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLineEdit" name="inputFileEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="inputFileBrowseButton">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="celSettingsGroupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>CEL/CL2 File Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="celClippedLabel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Clipped: </string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="celWidthLabel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Width: </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QWidget" name="celClippedWidget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <widget class="QRadioButton" name="celClippedYesRadioButton">
           <property name="text">
            <string>YES</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">celClippedButtonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="celClippedNoRadioButton">
           <property name="text">
            <string>NO</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">celClippedButtonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="celClippedAutoRadioButton">
           <property name="text">
            <string>Auto detect</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">celClippedButtonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>140</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
       <item row="1" column="1">
         <widget class="QWidget" name="celWidthWidget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="spacing">
               <number>4</number>
             </property>
             <property name="leftMargin">
               <number>4</number>
             </property>
             <property name="topMargin">
               <number>4</number>
             </property>
             <property name="rightMargin">
               <number>4</number>
             </property>
             <property name="bottomMargin">
               <number>4</number>
             </property>
             <item>
               <widget class="QLineEdit" name="celWidthEdit">
                 <property name="minimumSize">
                   <size>
                     <width>48</width>
                     <height>0</height>
                   </size>
                 </property>
                 <property name="maximumSize">
                   <size>
                     <width>48</width>
                     <height>16777215</height>
                   </size>
                 </property>
                 <property name="toolTip">
                   <string>Frame width</string>
                 </property>
               </widget>
             </item>
             <item>
               <spacer name="framesHorizontalSpacer_2">
                 <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                   <size>
                     <width>40</width>
                     <height>20</height>
                   </size>
                 </property>
               </spacer>
             </item>
           </layout>
         </widget>
       </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="openButtonsWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>345</width>
          <height>19</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="openButton">
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="openCancelButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="celClippedButtonGroup"/>
 </buttongroups>
</ui>
